exercises/binary-search-tree/test/test_binary_search_tree.c

```diff
@@ -0,0 +1,217 @@
+#include "vendor/unity.h"
+#include "../src/binary_search_tree.h"
+#include <stdlib.h>
+#include <stdbool.h>
+
+#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof(arr[0]))
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static void free_tree(node_t * tree)
+{
+   if (tree == NULL)
+      return;
+   free_tree(tree->left);
+   free_tree(tree->right);
+   free(tree);
+}
+
+void test_data_data_is_retained(void)
```

Comment:
Is purposely `data` repeated twice in this function name?

Reply:
After looking at as many as possible canonical-data.json I decided to stick to the following format when generating the test-functions:
```
   "test_" + property + description
```
which in this case led to the repeating 'data'.

Reply:
Great, thanks for the confirmation :+1: